src/Transport/Seam/TransportResourcesCreator.cs

```diff
@@ -17,7 +17,16 @@ public TransportResourcesCreator(ICreateTopology topologyCreator, ITopologySecti
 
         public async Task CreateQueueIfNecessary(QueueBindings queueBindings, string identity)
         {
-            if (resourcesCreated) return;
+            if (resourcesCreated)
+            {
+                return;
```

Comment:
Create queues only gets called when the installers are enabled. The assumption of the code is that although the implementation can be called multiple times the queue bindings will contain all the bindings to create and therefore we only create the topology once. I don't think we need to change this as part of the hotfix.